### ISSUE TITLE
[CI?] test_byte_align_set_dtype fails on 32-bit Linux (i686)

### ISSUE THREAD

**Issue by JeppeKlitgaard** [pyFFTW/pyFFTW#325]
This cropped up during the implementation of PR #323, but seems to be a bug rather than due to the CI implementation.

To get #323 through, testing of the 32-bit Linux builds has been temporarily suspended. This should be reenabled in `pyproject.toml` once this issue is fixed.

```py
___________________ ByteAlignTest.test_byte_align_set_dtype ____________________
self = <tests.test_pyfftw_nbyte_align.ByteAlignTest testMethod=test_byte_align_set_dtype>

    def test_byte_align_set_dtype(self):
        shape = (10,10)
        a = numpy.int16(numpy.random.randn(*shape)*16000)
        b = numpy.float64(numpy.random.randn(*shape))

        c = numpy.int8(numpy.random.randn(*shape)*255)
        # Test a few alignments
        for n in [None, 3, 7, 9, 16, 24, 23, 63, 64]:
            expected_alignment = get_expected_alignment(n)
            d = byte_align(a, dtype='float32', n=n)
>           self.assertTrue(d.ctypes.data % expected_alignment == 0)
E           AssertionError: False is not true
```

**Comment by hgomersall** [comment]
Do you know what alignment it's failing on? Is it consistent?

**Comment by hgomersall** [comment]
I'm wondering if non-word alignments are just meaningless.

**Comment by JeppeKlitgaard** [comment]
No, I don't know. The test could be updated to display which `n` it is failing on. I don't actually have a 32-bit Linux test setup, but one could possibly set it up a Docker instance.

I suspect it might be failing on 63 and/or 64, since it would make sense for these to potentially pass on 64-bit and not on 32-bit?

**Comment by hgomersall** [comment]
> No, I don't know. The test could be updated to display which `n` it is failing on. I don't actually have a 32-bit Linux test setup, but one could possibly set it up a Docker instance.

That's the problem!

**Comment by JeppeKlitgaard** [comment]
@hgomersall @grlee77 was this ever fixed? If I understand correctly we would just need to drop `63` and `64` as test cases for 32-bit systems to be able to re-enable 32-bit Linux testing again?
